### ISSUE TITLE
508-defect-1 [SCREENREADER]: STEM Standalone - Critical information is not available to the user with keyboard navigation

### ISSUE THREAD

**Issue by noahgelman** [department-of-veterans-affairs/va.gov-team#18254]
# [508-defect-1](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

Noah Gelman

## User Story or Problem Statement

Multiple critical text sections of the form which provides information to the user is not available to a user who relies on keyboard navigation and a screen reader to announce important information

## Details

Multiple important sections are not announced for users

## Acceptance Criteria

Add `tabindex="0"` to the following sections (`tabindex="0"` is already used in some places in the form)
- [ ]  Step 3 of 6 (Not qualified): "You must meet the above requirements to qualify for the scholarship" section is not announced
- [ ] Step 5 of 6: Direct Deposit Text block is not announced with keyboard navigation
- [ ] Step 5 of 6: "What if I don't have a bank account" expandable section content is not announced

## Steps to Recreate

https://staging.va.gov/education/other-va-education-benefits/stem-scholarship/apply-for-scholarship-form-22-10203/applicant/information

Navigate through the form

## WCAG or Vendor Guidance (optional)

https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-keyboard-operable.html

**Comment by elder-joseph** [comment]
Associated PR: https://github.com/department-of-veterans-affairs/vets-website/pull/15668

**Comment by delli-gatti-michael** [comment]
@noahgelman This is on staging and ready for review

**Comment by noahgelman** [comment]
@delli-gatti-michael Approved
